shared/actions/signup.js

```diff
@@ -57,6 +57,31 @@ function checkInviteCode (inviteCode: string): TypedAsyncAction<CheckInviteCode
   })
 }
 
+function requestAutoInvite (): TypedAsyncAction<CheckInviteCode | NavigateAppend | NavigateTo | SignupWaiting> {
+  return dispatch => new Promise((resolve, reject) => {
```

Comment:
@chrisnojima Just 'cause every other action in actions/signup does too and I wanted to leave the action style alone.  Tested that removing the promise works fine too.